components/Formatter.php

```diff
@@ -135,4 +136,16 @@ private function replaceCommentHeadlines($html)
     {
         return preg_replace('/<h\d+.*?>(.*?)<\\/h\d+>/i',"<p><strong>\\1</strong></p>", $html);
     }
+
+    /**
+     * @param string $html
+     *
+     * @return string
+     */
+    private function replaceImageUrlForProxy($html)
+    {
+        return preg_replace_callback('/(<img[^>]+?)src="(.+?)"/i', function($matches) {
```

Comment:
Instead of `"`, `'` or nothing could be used for attribute values. Correct regexp would be:

```php
return preg_replace_callback('/(<img[^>]+?)src=(?:["']([^"']+)["']|([^\s"'`=<>]+))/i', function($matches) {
```

Reply:
Done.